<launch>
  <arg name="video_device_num" default="0" />
  <arg name="video_device" default="/dev/video$(arg video_device_num)" />
  <arg name="image_width" default="640" />
  <arg name="image_height" default="480" />
  <arg name="loop_rate" default="60" />

  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device_num" value="$(arg video_device_num)" />
    <param name="video_device" value="$(arg video_device)" />
    <param name="image_width" value="$(arg image_width)" />
    <param name="image_height" value="$(arg image_height)"/>
    <param name="loop_rate" value="$(arg loop_rate)"/>

    <param name="pixel_format" value="mjpeg" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>
</launch>
